### ISSUE TITLE
Problem building WASM with tree-sitter-cli@0.19.5

### ISSUE THREAD

**Issue by char0n** [tree-sitter/tree-sitter#1201]
Hi all,

I'm having problem with tree-sitter-cli@0.19.5 and building the WASM file.

After running following command:
```sh
$ tree-sitter generate ./grammar.js && tree-sitter build-wasm
```
I get following result:
```sh
emcc was not found on PATH
```

When working with tree-sitter-cli@0.19.4 everything works as expected.

**Comment by ahlinc** [comment]
It's know fixed regression https://github.com/tree-sitter/tree-sitter/issues/1124#issuecomment-846579602 but there wasn't a release after the fix.

**Comment by char0n** [comment]
@ahlinc cool thanks, I didn't search in closed issues, otherwise I'd find #1124.

**Comment by ahlinc** [comment]
@maxbrunsfeld this issue is fixed, we can close it.
